MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -227,6 +236,11 @@ private void PlatformInitialize()
                     "MonoGame requires either ARB_framebuffer_object or EXT_framebuffer_object." +
                     "Try updating your graphics drivers.");
             }
+
+            // Force reseting states
+            this.BlendState.PlatformApplyState(this, true);
```

Comment:
I think it would be better to change these to `_actualBlendState`, etc. That will use the `GraphicsDevice`-specific instances of static state objects.

I can see why it works, as it is now - because it happens that we don't create any device-specific objects in `BlendState.OpenGL.cs`. But that might change in the future.